### DISCUSSION TITLE
How to detect the "stop" status on the rainbird station?

### DISCUSSION THREAD

**Discussion by Eckart0** [allenporter/pyrainbird#64] [General]
I can't find a var representig the "stop"-status at my rainbird station. Did someone find it? 
"rainbird.X.device.irrigation.active" seems to be always true.
Regards, Eckart

**Comment by allenporter** [comment]
Hi, Assuming I understand your question, my impression is you can check if a zone is active by checking the `get_zone_states` call which has state about each zone.
